src/useCases/hasUserAccessToUrl.ts

```diff
@@ -3,19 +3,22 @@ import getUserServiceAccess from "./getUserServiceAccess"
 
 const userManagementUrlExpression = /^\/users\/users.*/
 const bichardUrlExpression = /^\/bichard-ui.*/
+const newBichardUrlExpression = /^\/bichard\/.*|^\/bichard$/
```

Comment:
FYI - you can add a `?` after a character to make it optional, which might be useful for the trailing slash